unattend/OxidePrepBaseImage.ps1

```diff
@@ -36,7 +106,28 @@ do {
 
 #region Enable SSH
 Write-Host "Enabling SSH"
-Add-WindowsCapability -Online -Name OpenSSH.Server~~~~0.0.1.0
+
+# The easiest way to install OpenSSH is to install the relevant Windows
+# capability, but this only exists in-box on Windows Server 2019 and later.
+# Server 2016 recognizes the capability name, but enabling it doesn't actually
+# install the sshd service. Try to detect both of these cases.
+Add-WindowsCapability -Online -Name OpenSSH.Server~~~~0.0.1.0 -ErrorAction SilentlyContinue
+if ($?) {
+    $sshCap = Get-Service -Name sshd -ErrorAction SilentlyContinue
+}
+
+# If either of the last two commands produced an error, fall back to trying to
+# pull the latest release down from GitHub and trying to install it manually.
+if ($?) {
+    Write-Host "SSH service installed via Add-WindowsCapability"
+} else {
+    Write-Host "SSH capability not present in image, will download from GitHub"
+    $sshPath = "C:\Windows\Temp\OpenSSH-Win64.zip"
+    RetryWithBackoff -ScriptBlock { DownloadLatestSshArchive -ArchivePath $sshPath }
+    DownloadLatestSshArchive -ArchivePath $sshPath
```

Comment:
Leftover line?

Reply:
just testing for idempotency, nothing to see here

(Good catch, thanks!)